src/Tools/dotnet-dump/Analyzer.cs

```diff
@@ -102,6 +104,7 @@ or NotSupportedException
                 contextService.SetCurrentTarget(target);
 
                 // Automatically enable symbol server support, default cache and search for symbols in the dump directory
+                HttpClient.DefaultProxy.Credentials = CredentialCache.DefaultCredentials;
```

Comment:
This change should be in the SymbolService or the HttpSymbolStore.

Reply:
I will move it there. You have seen the consider actions why  I placed it here?